omero/sysadmins/public.rst

```diff
@@ -60,6 +60,15 @@ To set this up on your OMERO.web installation:
 
      $ bin/omero config set omero.web.public.password '<password>'
 
+- By default the public user is only allowed to perform GET requests. This means
```

Comment:
I moved this section above the url_filter section as @will-moore suggested. But I'm a bit worried, that it now actually looks like something the user is supposed to do, as part of setting up the public access.